.gitignore

```diff
@@ -116,16 +116,14 @@ output.ipynb
 # don't save any data
 classification/data/*
 data/
+!contrib/action_recognition/r2p1d/**
 
 # don't save .swp files
 *.swp
 
 # don't save .csv files
 *.csv
 
-# don't save data dir
-data
```

Comment:
why remove this?

Reply:
@jiata Thank you for the review! For this, we already have `data/` above at line 118